vernac/vernacentries.ml

```diff
@@ -1013,9 +1013,9 @@ let vernac_include l =
 
 (* Sections *)
 
-let vernac_begin_section ({v=id} as lid) =
+let vernac_begin_section ~poly ({v=id} as lid) =
   Dumpglob.dump_definition lid true "sec";
-  Lib.open_section id
+  Lib.open_section ~poly id
```

Comment:
I feel like we should also do `Local Set ~poly` so that we don't have to repeat it inside the section.

Reply:
See the answer to @ejgallego below.